opal/mca/base/mca_base_var.c

```diff
@@ -100,7 +101,8 @@ const size_t ompi_var_type_sizes[] = {
     sizeof (char),
     sizeof (char),
     sizeof (bool),
-    sizeof (double)
+    sizeof (double),
+    sizeof (long),
```

Comment:
Please remove the trailing comma -- it generates compiler warnings.

Reply:
Odd that is complains about that. The trailing comma is allowed by C99 and it is required that the compiler support it. Do you have the warning message handy?

Reply:
See C99 6.7.8.

Reply:
Ah, if this is allowable via C99, then I retract my objection.

Reply:
I will still look into the message as I am curious what is going on. We use trailing commas all over the place.

Reply:
Don't spend any time on it.  I am probably remembering something with an old compiler and/or before we enforced C99-specific compiler options.